src/executables/uber-price.js

```diff
@@ -25,19 +25,25 @@ const symbolService = new SymbolService();
 program
   .option('-s, --start <start>', 'specify start address')
   .option('-e, --end <end>', 'specify end address')
-  .option('-u, --unit <unit>', 'specify distance unit')
+  .option('-u, --unit [unit]', 'specify distance unit')
   .parse(process.argv);
 
-try {
-  service.executePriceEstimates(program.start, program.end, program.unit)
-    .then(table => console.log(table))
-    .catch((e) => {
-      if (isDistanceExceededError(e)) {
-        console.log(`Maximum distance of ${symbolService.getMaximumDistanceSymbol()}  miles exceeded between start address: ${program.start} and end address: ${program.end}`);
-      } else {
-        console.error('Could not get price estimates');
-      }
-    });
-} catch (e) {
-  console.error('Could not get price estimates');
+
+if (typeof program.start !== 'string' && typeof program.end !== 'string') {
```

Comment:
It's more a catch if they're ever anything *but* strings. It may be unnecessary.